NEWS.md

```diff
@@ -8,6 +8,18 @@
 - RStudio no longer highlights `\[ \]` and `\( \)` Mathjax equations; prefer `$$ $$` and `$ $` instead (#12862)
 - Added cmake option to build RStudio without the check for updates feature (#13236)
 - Allow choosing R from non-standard location at startup (#14180; Windows Desktop)
+- Updated Ace to version 1.28. (#13708)
```

Comment:
These NEWS updates look unintentional?

Reply:
Thanks! For catching this. I'll fix up my rebase.